recipes/zlib-ng/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package LANGUAGES C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(ZLIB REQUIRED)
+
+add_executable(${PROJECT_NAME} enough.c)
+target_link_libraries(${PROJECT_NAME} ZLIB::ZLIB)
```

Comment:
This does not match the upstream behavoir https://github.com/zlib-ng/zlib-ng/blob/ac18b0c35c8cf56bcc2531e100372f654390923b/CMakeLists.txt#L902

This commit did something very similar https://github.com/conan-io/conan-center-index/commit/f42348714f2b877c0fa7919cddf48e804dbc2971